pkg/common/configs/configvalidator.go

```diff
@@ -509,19 +509,21 @@ func checkLimit(limit Limit, existedUserName map[string]bool, existedGroupName m
 				zap.Error(err))
 			return err
 		}
+		if resources.IsZero(limitResource) {
+			return fmt.Errorf("invalid MaxResources in '%s' limit, all resource limits are zero", limit.Limit)
```

Comment:
invalid MaxResources in '%s' limit, all resource limits are zero ->
MaxResources is zero in '%s' limit, all resource types are zero

Reply:
Updated it. Thank you.